k8s/cmd/commands/delete/delete_vineyardd.go

```diff
@@ -39,9 +39,10 @@ var deleteVineyarddCmd = &cobra.Command{
 	Short:   "Delete the vineyardd cluster from kubernetes",
 	Example: deleteVineyarddExample,
 	Run: func(cmd *cobra.Command, args []string) {
-		util.AssertNoArgs(cmd, args)
+		util.Check(cmd, args)
 		client := util.KubernetesClient()
 
+		log.Debug("deleting Vineyardd cr")
```

Comment:
"deleting Vineyardd cr" -> "deleting Vineyardd cluster"

As you state "delete the vineyard cluster ..." in help message.